src/browser/crypto.ts

```diff
@@ -0,0 +1,13 @@
+// Note: this polyfill doesn't include `crypto.timingSafeEqual()`. The browser
+// never runs a BTP server, so it doesn't need to compare tokens.
+
+const { crypto } = window
+
+export function randomBytes (
+  size: number,
+  callback: (err: Error | null, buf: Buffer) => void
+): void {
+  const randArray = new Uint8Array(size)
+  const randValues = crypto.getRandomValues(randArray)
+  callback(null, Buffer.from(randValues))
```

Comment:
@sentientwaffle I dont think Buffer is supported in the browser. Are we going to assume that people are going to be using webpack?

Reply:
Yes, I'm assuming that it is built with webpack (or, in general, that all the necessary libraries are polyfilled).